.github/CONTRIBUTING.md

```diff
@@ -154,15 +154,158 @@ the repo such that they can get coding -->
+
+_Example #3: using code snippets to explain what and how_
+
+When reporting a bug or similar do not write something along the lines of: 
+> Whenever i use a `kirby-button` with attention level 1 on the frontpage, a warning is thrown
+
+Specify exactly where and how the button is inserted with a code snippet, along with exact details of the warning and where it can be found:
+> Inserting a `kirby-button` with attention level 1 as shown in the code-snippet, results in the following warning being logged in the browser console: `Warning goes here`.
+
+The latter is pure maintainer poetry. 
+
+These are but three examples of how issues can be specific and unambigious. But always keep in the back of your mind how you can be more specific and unambigious when writing issues. 
+
+#### The good issue follows the template 
+<!-- - Follow the templates! - they are as is to help you remember which information we're looking for -->
+Do not delete the template when creating an issue. Use it. 
+
+The issue templates exists to ease collaboration by providing structure and reminding you of which information other contributors are looking for. 
+They also provide helpful checklists that helps you follow [the process (INSERT PROPER LINK HERE)](#the-process) and lets others know how far you are. 
+
+If you experience that no template lives up to your specific need; well there is a good opportunity to contribute. Create an issue for a new template to be created! 
+
+#### The good issue has a good title
+<!-- - Spend some time on your titles 
+  - Make them alluring. 
+  - Short without too much detail 
+  - The issues are viewed in some sort of list view 80% of the time. -->
+As issues are viewed in some sort of list view 80% of the time (or so) the title is the part being read most often. 
+It can therefore be a good idea to spend some time formulating one. 
+
+A good title is short without too much detail. Details should be presented in the actual description. 
+
+If possible try to make the title compelling. While an open-source repository is not a newspaper where readers must be enganged at any cost, you still want to catch the interest of contributors. 
+
+The following title: 
+> Input causes error when you type over 50 characters and press enter 
+
+could be improved by writing: 
+> Input error on 50-character submit
+
+#### The good issue is easily readable
+<!-- - Readability 
+- Easily understandable 
+  - No internal lingo 
+- Proof-read --> 
+
+There are MANY ways to improve the readability of a text. The following is just some ways to do it, that we think can add value when writing an issue:
+- Using proper punctuation (to the best of your abilities at least. We are not experts at ,it either)
+- Spending some time proof-reading the issue before it is submitted. 
+- Using available formatting such as lists and code blocks to aid easy skimming. 
+- Explaining the abbrevations being used 
```

Comment:
I suggest you use _imperative mood_ here - maybe even use it in general within list items:

* Use proper punctuation [...]
* Spend some time proof-reading [...]
* Use available formatting [...]
* Explain the abbrevations [...]

Reply:
Should be solved by 6ede0331559a38a46246aa6f703ae4b697ba1ae9